### PR TITLE
Added SDANN and SDNNIDX hrv metric

### PR DESCRIPTION
**Pull request by leifwar**

### CODE REVIEW

File: src/Metrics/HrvMetrics.cpp [hunk 1]
```diff
@@ -108,7 +109,8 @@ class avnn : public RideMetric {
 
     void compute(RideItem *item, Specification, const QHash<QString,RideMetric*> &) {
 
-        double total, count;
+        double total;
+        int count;
```
Comment (liversedge) [inline src/Metrics/HrvMetrics.cpp]: Why are you doing this?
You can store integers in a double...

**Comment by liversedge** [comment]
RideMetric::count needs to be a double.

**Comment by leifwar** [comment]
I was not thinking. Just saw the counter and thought that it will always be an integer. Have updated.

**Comment by leifwar** [comment]
Do you have a place/channel for make comments or asking questions to some specific code?

**Comment by liversedge** [comment]
We have a slack I will open a new channel for developer chat and invite you.
What is your email address?

**Comment by leifwar** [comment]
leif.warland@getmail.no

**Pull request merged**